s3-encryption/client.md

```diff
@@ -0,0 +1,142 @@
+[//]: # "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved."
+[//]: # "SPDX-License-Identifier: CC-BY-SA-4.0"
+
+# Client
+
+## Version
+
+0.1.0
+
+### Changelog
+
+- 0.1.0
+  - Initial
+
+## Definitions
+
+### Conventions used in this document
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL"
+in this document are to be interpreted as described in [RFC2119](https://tools.ietf.org/html/rfc2119).
+
+### Key Material
+
+An AES or RSA key, or a valid AWS KMS key identifier.
```

Comment:
maybe a place for it here, but what format do we expect these in? just bytes? what format does the rsa public key need to be in? der/pem?

Reply:
good shout, i'm going to defer that to the specific keyrings.